src/components/Popups/Popups.tsx

```diff
@@ -28,7 +23,7 @@ export function useShowPopup(): ShowPopupContextType {
 export function ShowPopupProvider({
   children,
 }: {
-  children: JSX.Element;
+  children: JSX.Element[] | JSX.Element;
 }): JSX.Element {
   const [visible, showPopup] = useState(false);
```

Comment:
Can we also use useBooleanState here?

Reply:
I considered that as well, and decided that I’ll have to map values to names in the next line, and this would not be elegant